prov/efa/src/efa_ep.c

```diff
@@ -622,68 +615,64 @@ static struct fi_ops_atomic efa_ep_atomic_ops = {
 	.compwritevalid = fi_no_atomic_compwritevalid,
 };
 
-int efa_ep_open(struct fid_domain *domain_fid, struct fi_info *info,
```

Comment:
since you modified it, could u add a docstring.

Reply:
I plan to refactor this function next.

I will move the rdm part of this function to `rxr_endpoing`, then rename it`efa_dgram_ep_open`. So I would prefer to  add docstring at that time.